src/super_gradients/training/processing/processing.py

```diff
@@ -455,6 +455,87 @@ def default_dekr_coco_processing_params() -> dict:
     return params
 
 
+def default_yolo_nas_pose_coco_processing_params() -> dict:
+    """Processing parameters commonly used for training DEKR on COCO dataset."""
+
+    image_processor = ComposeProcessing(
+        [
+            ReverseImageChannels(),
+            KeypointsLongestMaxSizeRescale(output_shape=(640, 640)),
+            KeypointsBottomRightPadding(output_shape=(640, 640), pad_value=127),
+            StandardizeImage(max_value=255.0),
+            NormalizeImage(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+            ImagePermute(permutation=(2, 0, 1)),
+        ]
+    )
+
+    edge_links = [
+        [0, 1],
```

Comment:
Yes, they are defined in `coco_pose_dataset_params`, however the goal of having them defined here is to provide a default params if they are missing in the checkpoint. Which is the case for your DEKR & YOLO-NAS checkpoints, since they were trained before we introduced Predict API for PE.